skymp5-server/cpp/addon/main.cc

```diff
@@ -360,6 +364,14 @@ ScampServer::ScampServer(const Napi::CallbackInfo& info)
       }
     }
 
+    if (serverSettings["lang"] == nullptr) {
+      serverSettings["lang"] = "english";
+    }
+
+    logger->info("Run localization provider");
+    localizationProvider = std::make_shared<LocalizationProvider>(
+      serverSettings["dataDir"], serverSettings["lang"]);
+
```

Comment:
I think that we should handle variant when this functionality is not needed. 
Maybe it's better to detect which languages we have onboard, so we will need some option like "useLocalizationProvider" with boolean value.
For now just make your new code not terminating initialization process if there're no strings files.
Something like:
```suggestion
    if (serverSettings["lang"] != nullptr) {
      logger->info("Run localization provider");
      localizationProvider = std::make_shared<LocalizationProvider>(
        serverSettings["dataDir"], serverSettings["lang"]);
    }
```